guides/pages/how-to-synchronize-your-yjs-storage-data-to-a-planetscale-mysql-database.mdx

```diff
@@ -0,0 +1,584 @@
+---
+meta:
+  title:
+    "How to synchronize your Yjs Storage data to a PlanetScale MySQL database"
+  description:
+    "Learn how to automatically update your PlanetScale MySQL database with
+    changes from your Yjs application."
+---
+
+Liveblocks allows you to build collaborative applications with
+[Yjs](https://yjs.dev/), which is tightly integrated into our infrastructure.
+Using our Webhooks and REST API, you can then retrieve the Yjs document data as
+it changes, and store it in your database.
+
+## What we’re building
+
+In this guide we’ll be linking a Yjs application up to a
+[PlanetScale](https://planetscale.com/) MySQL database so that Yjs document data
+is automatically synchronized. This is enabled through the following:
+
+- [YDocUpdated](/docs/platform/webhooks#YDocUpdatedEvent) Webhook event
+- [Get Yjs Document](/docs/api-reference/rest-api-endpoints#get-rooms-roomId-ydoc)
+  REST API
+
+<Banner title="Have a project ready">
+
+This guide assumes you already have a Liveblocks Yjs project set up. If haven’t
+already got one, you can open our getting started page, select a
+[text editor](/docs/get-started/text-editor) or
+[code editor](/docs/get-started/code-editor), both of which use Yjs, and come
+back after you’re set up.
+
+</Banner>
+
+### This specific webhook is throttled
+
+Note that the [YDocUpdated](/docs/platform/webhooks#YDocUpdatedEvent) webhook
+event is throttled at a rate of _once every 5 minutes_. This is because Yjs can
```

Comment:
Not related to your guide but 5 minutes feels too long to me. Are there any reasons we couldn't run this more frequently like every 5-10 seconds?

Reply:
Don't know really. 5 mins does feel too long.